db/seeds.rb

```diff
@@ -7,7 +7,7 @@
   gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
   User.create!(
     name: "Test user",
-    permissions: ["signin", "GDS Admin", "GDS Editor", "Managing Editor", "Export data"],
+    permissions: ["signin", "GDS Admin", "GDS Editor", "Managing Editor", "Export data", "Sidekiq Admin"],
```

Comment:
should we just add this for the Test User for good? I feel like it will continue to be useful as we continue to use Sidekiq...

Reply:
Yeah, I can't see the harm 👍